src/Interactive/EditorFeatures/Core/Extensibility/Interactive/NuGetPackageResolverImpl.cs

```diff
@@ -0,0 +1,170 @@
+            {
+                return default(ImmutableArray<string>);
+            }
+
+            var tempPath = PathUtilities.CombineAbsoluteAndRelativePaths(Path.GetTempPath(), Guid.NewGuid().ToString("D"));
+            var tempDir = Directory.CreateDirectory(tempPath);
+            try
+            {
+                // Create project.json.
+                var projectJson = PathUtilities.CombineAbsoluteAndRelativePaths(tempPath, "project.json");
+                using (var stream = File.OpenWrite(projectJson))
+                using (var writer = new StreamWriter(stream))
+                {
+                    WriteProjectJson(writer, packageName, packageVersion);
+                }
+
+                // Run "nuget.exe restore project.json" to generate project.lock.json.
+                NuGetRestore(projectJson);
+
+                // Read the references from project.lock.json.
+                var projectLockJson = PathUtilities.CombineAbsoluteAndRelativePaths(tempPath, "project.lock.json");
+                using (var stream = File.OpenRead(projectLockJson))
+                using (var reader = new StreamReader(stream))
+                {
+                    return ReadProjectLockJson(_packagesDirectory, reader);
+                }
+            }
+            finally
+            {
+                tempDir.Delete(recursive: true);
+            }
+        }
+
+        /// <summary>
+        /// Syntax is "id/version", matching references in project.lock.json.
+        /// </summary>
+        internal static bool ParsePackageReference(string reference, out string name, out string version)
+        {
+            int separator = reference.IndexOf('/');
+            int n = reference.Length;
+            if ((separator > 0) &&
+                (separator < n - 1) &&
+                (reference.IndexOf('/', separator + 1) < 0))
+            {
+                name = reference.Substring(0, separator);
+                version = reference.Substring(separator + 1);
+                return true;
+            }
+            name = null;
+            version = null;
+            return false;
+        }
+
+        /// <summary>
+        /// Generate a project.json file with the packages as "dependencies".
+        /// </summary>
+        internal static void WriteProjectJson(TextWriter writer, string packageName, string packageVersion)
+        {
+            using (var jsonWriter = new JsonTextWriter(writer))
+            {
+                jsonWriter.Formatting = Newtonsoft.Json.Formatting.Indented;
+                jsonWriter.WriteStartObject();
+                jsonWriter.WritePropertyName("dependencies");
+                jsonWriter.WriteStartObject();
+                jsonWriter.WritePropertyName(packageName, escape: true);
+                jsonWriter.WriteValue(packageVersion);
+                jsonWriter.WriteEndObject();
+                jsonWriter.WriteEndObject();
```

Comment:
You should just be writing out framework net45 or whatever you're minimum target is. Otherwise NuGet won't know which asset to restore in some cases and will likely choose very wrong. (I'm actually shocked it let you restore at all without one.)